<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Logo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Yu Gothic&quot;;
background:
    white;
border-radius:
    20px;
border-style:
    solid;
border-width:
    3px;
border-color: 
black</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Icons/Upload.jpg</normaloff>C:/Icons/Upload.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Yu Gothic&quot;;
background:
    white;
border-radius:
    20px;
border-style:
    solid;
border-width:
    3px;
border-color: 
black</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Icons/Clear.jpg</normaloff>C:/Icons/Clear.jpg</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="RotateOptions">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Rotate Options </string>
    </property>
    <widget class="QRadioButton" name="R90">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>31</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>90° Degree</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="R270">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>83</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>270° Degree</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Apply2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="R180">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>57</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>180° Degree</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FlipOptions">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Flip Options </string>
    </property>
    <widget class="QRadioButton" name="AA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontally</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="VV">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertically</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Apply">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DkLgOptions">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Darken/Lighten Options</string>
    </property>
    <widget class="QRadioButton" name="DK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Darken</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="LG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lighten</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Apply3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ShrinkOptions">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Shrink Options</string>
    </property>
    <widget class="QRadioButton" name="S12">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>31</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1/2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="S14">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>83</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1/4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Apply4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="S13">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>57</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1/3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EnlargeOptions">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Enlarge Options</string>
    </property>
    <widget class="QRadioButton" name="A1">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>21</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1st Quarter</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="A3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>73</y>
       <width>83</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3rd Quarter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Apply5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="A2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>47</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2nd Quarter</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="A4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>99</y>
       <width>83</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4th Quarter</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Before">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>340</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="After">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);
self.label_1.setStyleSheet(&quot;border: 1px solid black;&quot;)</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="MirrorOptions">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Mirror Options</string>
    </property>
    <widget class="QRadioButton" name="M1">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>21</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>M. Left</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="M3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>73</y>
       <width>83</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>M. Upper</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Apply6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="M2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>47</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>M. Right</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="M4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>99</y>
       <width>83</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>M. Lower</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Shuffle Options</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:2px solid gray;
border-radius:10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter the Sequence</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Yu Gothic&quot;;
background:
    white;
border-radius:
    20px;
border-style:
    solid;
border-width:
    3px;
border-color: 
black</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Icons/Save.jpg</normaloff>C:/Icons/Save.jpg</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>201</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="Filter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>B/W</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Filter_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Invert</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Filter_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Flip</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Filter_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Rotate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Filter_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Shrink</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Filter_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Filter_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Shuffle</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Filter_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Yu Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Dark / Light</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Filter_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Blur</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Filter_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Yu Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Detect Edge</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="Filter_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Yu Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Enlarge</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="Filter_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">;font: 12pt &quot;Yu Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Mirror</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>RotateOptions</zorder>
   <zorder>Logo</zorder>
   <zorder>Load</zorder>
   <zorder>Clear</zorder>
   <zorder>FlipOptions</zorder>
   <zorder>DkLgOptions</zorder>
   <zorder>ShrinkOptions</zorder>
   <zorder>EnlargeOptions</zorder>
   <zorder>Before</zorder>
   <zorder>After</zorder>
   <zorder>MirrorOptions</zorder>
   <zorder>groupBox</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
